aptos-move/e2e-move-tests/src/tests/scripts.rs

```diff
@@ -82,341 +80,3 @@ fn read_coin(h: &MoveHarness, account: &AccountAddress) -> u64 {
         .unwrap()
         .coin()
 }
-
-#[test]
-fn test_two_to_two_transfer_gas_payer() {
-    let mut h = MoveHarness::new_with_features(vec![FeatureFlag::GAS_PAYER_ENABLED], vec![]);
-
-    let alice = h.new_account_at(AccountAddress::from_hex_literal("0xa11ce").unwrap());
```

Comment:
I guess we moved these?

Reply:
Yes they got moved to fee_payer. These shouldn't have been here in the first place. scripts.rs seems to getting a lot of random tests that don't really belong